README.markdown

```diff
@@ -1,5 +1,15 @@
 # Meilisearch Documentation
```

Comment:
```suggestion
<p align="center">
  <img src="assets/meilisearch-logo-light.svg?sanitize=true#gh-light-mode-only">
  <img src="assets/meilisearch-logo-dark.svg?sanitize=true#gh-dark-mode-only">
</p>

<h1 align="center">Meilisearch Documentation</h1>
```

You cam add the logos from [here](https://github.com/meilisearch/meilisearch/tree/main/assets)